api/customAttributes.js

```diff
@@ -93,7 +93,22 @@ module.exports = function(app) {
       retObj.stem = id.length === 6 ? stems.includes(id) ? "Stem Major" : false : "Contains Stem Majors"
     }
     else if (dimension === "University") {
+      const loadJSON = require("../utils/loadJSON");
+      const universitySimilar = loadJSON("/static/data/similar_universities_with_dist.json");
+
+      const similarID = universitySimilar[id.toString()] ? universitySimilar[id.toString()].map(d => d.university) : false
```

Comment:
AFAIK you should not have to cast `toString` here, because I think all Object keys in JavaScript are coerced to strings regardless.

Additionally, I think we might prefer the default state for `similarID` to be `[]` instead of `false`. I think it would make error catching easier in the frontend, and simplify your `map` logic below (you don't need to check if `similarID` exists, it will always be an Array.